### PR TITLE
Bipartite graph

### PR DESCRIPTION
**Pull request by vanshikachokhani**
### Changes made
I have written bfs traversal approach to check if graph is bipartite in C++.
This is a contribution as a part of GSSoC21.

I have tested the code and it works fine for every test case and included some sample test cases and also the algorithm.

### CODE REVIEW

File: Code/C++/check_if_bipartite_graph.cpp [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    2 3
+    3 4
+    4 0
+Output:
+    The graph is not Bipartite
+
+Time Complexity : O(V + E)  
+Space Complexity : O(V)     
+where V is the number of vertices and E is the no of edges
+*/
```
Comment (raksha009) [inline Code/C++/check_if_bipartite_graph.cpp:103]: Add a new line here.

File: Code/C++/check_if_bipartite_graph.cpp [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+    if(checkBipartite(adjacent, vertices))
+    cout<<"The graph is Bipartite"<<endl;
+    else
+    cout<<"The graph is not Bipartite"<<endl; 
+
+    return 0;
+}
+
+/*
```
Comment (raksha009) [inline Code/C++/check_if_bipartite_graph.cpp:78]:
```suggestion
/*
Test Cases
```

File: Code/C++/check_if_bipartite_graph.cpp [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+Check if a graph is bipartite graph.
+
+A Bipartite graph is a graph whose vertcies can be divided into independent sets such that for every edge(u,v) u and v belong to different set i.e all edges should be from one set to another.
+A bipartite graph is possible if the graph coloring is possible using two colors such that vertices in a set are colored with same color.
+*/
```
Comment (raksha009) [inline Code/C++/check_if_bipartite_graph.cpp:6]: Merge these 2 into a single multiline comment

**Review by raksha009 — CHANGES_REQUESTED** [review]

File: Code/C++/check_if_bipartite_graph.cpp (hunk 3 above)
Comment (vanshikachokhani) [reply Code/C++/check_if_bipartite_graph.cpp:5]: @raksha009 I have done the changes. Please review it.

**Review by raksha009 — APPROVED** [review]

**Pull request merged**